.github/workflows/release-please.yml

```diff
@@ -16,27 +16,21 @@ jobs:
         id: release
         with:
           token: ${{ steps.get-token.outputs.token }}
-          command: manifest
-      - uses: actions/checkout@v3
-        if: ${{ steps.release.outputs.releases_created }}
-      - uses: actions/setup-node@v3
+          release-type: node
+          package-name: '@netlify/plugin-nextjs'
+      - uses: actions/checkout@v2
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v2
         with:
           node-version: '*'
           cache: 'npm'
           check-latest: true
           registry-url: 'https://registry.npmjs.org'
-        if: ${{ steps.release.outputs.releases_created }}
+        if: ${{ steps.release.outputs.release_created }}
```

Comment:
I'm copying this from https://github.com/netlify/vite-plugin-netlify-edge @ascorbic, so I'm assuming it's `release_created` and not `releases_created`?

Reply:
Yes, it's `releases_created` if using a manifest to handle releases, and `release_created` if using the default

Reply:
Sounds good. I removed the manifest in this PR as it doesn't seem to be required based on https://github.com/netlify/vite-plugin-netlify-edge, so this should be good to merge then.